ddp-workspace/projects/ddp-sdk/src/public-api.ts

```diff
@@ -63,10 +63,12 @@ export * from './lib/services/renewSessionNotifier.service';
 export * from './lib/services/scriptLoader.service';
 export * from './lib/services/serviceAgents/governedParticipantsServiceAgent.service';
 export * from './lib/services/languageService.service';
+export * from './lib/services/serviceAgents/invitationServiceAgent.service';
 
 export * from './lib/components/login/auth0-code-callback.component';
 export * from './lib/components/address/addressEmbedded.component';
 export * from './lib/components/activityForm/activity.component';
+export * from './lib/directives/upperCaseInputDirective.directive';
```

Comment:
I moved upperCaseInputDirective in the directives folder.

Reply:
@MocanaAtBroad, why do you that's a bad idea?

Reply:
Ah, we also can remove this export, it is not necessary, importing in the SDK module is enough.

Reply:
Sorry! I meant to give you a thumbs up! 👍 
Actually, I think I can remove it now as it is no longer used now that we have directive that formats the code exactly as needed, including upper casing.
It is likely that the invitation code input box will become its own components as we want to use same idea in future projects and there will be no need to export the directives.

Reply:
Though as you mentioned, the formatter directive is not there and appears to work OK. So now I am confused about this public-api.ts thing and will have to do additional reading. I thought you had to import module no matter what if you want to use stuff.
I am going to adjust this anyways just to export the formatter directive as that is the only one we are using outside and will figure out what magic is going on here.

Reply:
Ah, no problems :) 
We also use `LazyLoadResourcesDirective` outside of SDK.